ASIM/schemas/entities/ASimUser.yaml

```diff
@@ -57,3 +57,15 @@ Fields:
   Class: Optional
   Description: The unique ID of the sign-in session of the user.
   Example: 102pTUgC3p8RIqHvzxLCHnFlg
+
+- Name: <<Role>>Domain
+  Type: string
+  Class: Recommended
+  Description: The actor's domain.
+
+- Name: <<Role>>DomainType
+  Type: string
+  Class: Conditional
+  Logical type: Enumerated
+  Description: The actor's domain type.
+  Follows: <<Role>>Domain
```

Comment:
Can we set property ForRoles? with Actor, Target
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/d3e41163-1896-4a29-96a2-3593a3f844f8)

Reply:
Changed  the <<Role>> to Target because we also have the ASimActor.yaml